### DISCUSSION TITLE
无法挂载pikpak

### DISCUSSION THREAD

**Discussion by luzhu9** [AlistGo/alist#3156] [Q&A]
报错：Failed init storage: Post "https://user.mypikpak.com/v1/auth/signin": dial tcp ************3:443: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond.
clash开启tun之后显示tls握手失败
不知道该怎么办

**Comment by anwen-anyi** [comment]
你clash怎么写的？ clash的端口是7890默认你不改的话这样就可以了
```bash
export https_proxy=http://127.0.0.1:7890 http_proxy=http://127.0.0.1:7890 all_proxy=socks5://127.0.0.1:7890
```

**Reply by limbia** [reply]
是pac内容嘛 我平时也用unblocknetease和hyplayer 所以改成了这个 
function FindProxyForURL(url, host) {
  if (host == 'music.163.com' || host == 'interface.music.163.com' || host == 'interface3.music.163.com' || host == 'apm.music.163.com' || host == 'apm3.music.163.com' || host == 'musicupload.netease.com') {
		return 'PROXY 127.0.0.1:8080'
	}
  return "PROXY 127.0.0.1:%mixed-port%; SOCKS5 127.0.0.1:%mixed-port%; DIRECT;"
}

是在这改嘛? 我改成你这个好像没用

**Comment by limbia** [comment]
我也是一样的 但报错是长这样
failed get objs: failed to list objs: Get "https://api-drive.mypikpak.com/drive/v1/files?filters=%7B%22phase%22%3A%7B%22eq%22%3A%22PHASE_TYPE_COMPLETE%22%7D%2C%22trashed%22%3A%7B%22eq%22%3Afalse%7D%7D&limit=100&page_token=&parent_id=&thumbnail_size=SIZE_LARGE&with_audit=true": dial tcp ************41:443: connectex: No connection could be made because the target machine actively refused it.

**Reply by fengxiansen9** [reply]
兄弟们这个问题解决了吗？我的也是报错

**Reply by ztsun12345** [reply]
试一下TUN模式的全局代理

**Reply by littleLice** [reply]
#### 我也报错：
无法显示目录下文件，请刷新重试: Post "https://user.mypikpak.com/v1/auth/token": EOF

**Comment by HiGodfather** [comment]
我也报错：

failed get enabled storages: Post "https://user.mypikpak.com/v1/auth/token": EOF

**Comment by curryhendry** [comment]
我是过一段时间添加能成功，一周以内就会提示这个，什么全局之类的都不好使，不是代理的事儿吧。能连上也没有全局。
